proto/rollup/v1/msg.proto

```diff
@@ -14,5 +20,26 @@ message ApplyL1TxsRequest {
   repeated bytes tx_bytes = 1;
 }
 
-message ApplyL1TxsResponse {
+message ApplyL1TxsResponse {}
+
+// MsgInitiateWithdrawal defines a message for all L2 withdrawal txs
+message InitiateWithdrawalRequest {
+  option (cosmos.msg.v1.signer) = "sender";
+  // The cosmos address of the user who wants to withdraw from L2.
+  string sender = 1;
```

Comment:
is there a more descriptive type for addresses?

Reply:
Looks like I can add `[(cosmos_proto.scalar) = "cosmos.AddressString"];` for an auto conversion, I'll add this in